src/Behat/Behat/Context/Cli/InteractiveContextIdentifier.php

```diff
@@ -70,7 +70,7 @@ public function guessTargetContextClass(ContextEnvironment $environment)
 
         $message = $this->translator->trans('snippet_context_choice', array('%1%' => $suiteName), 'output');
         $choices = array_values(array_merge(array('None'), $contextClasses));
-        $default = current($contextClasses);
+        $default = 1;
```

Comment:
that's weird to me. The first choice is 0, not 1

Reply:
It used to select the first context class from `$contextClasses`, which is the second element of `$choices`. The first choice is `None`, and that's not the default we want.